src/components/Alert/Alert.module.css

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2023  New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/* TODO: Review entire file for semantic token definiton */
+
+.alert {
+  display: flex;
+  align-items: center;
```

Comment:
(More of a design comment, but...) Vertically centering the close button means it won't be in a consistent position (depending on longer/shorter content). This would be super annoying if these are used as dialogs

Reply:
Makes a lot of sense! Will update that